services/ChoresService.php

```diff
@@ -187,11 +187,13 @@ public function TrackChore(int $choreId, string $trackedTime, $doneBy = GROCY_US
 			}
 		}
 
+		$estimatedExecutionTime = $this->getDatabase()->chores_current()->where('chore_id', $choreId)->min('next_estimated_execution_time');
```

Comment:
Why is `min('next_estimated_execution_time')` needed here? `chores_current` already only contains one row per chore...

Reply:
This has been in the begin a lazy copy/paste from function GetChoreDetails.
I later noticed the min and was wondering a bit about it and thought it will not do any harm and better be on the safe side, there might be reason for the "min"

Reply:
> if you are happy about contributions like this

Please don't get me wrong, but reviewing a contributed copy & paste job, without really understanding what's happening, takes probably fourth the time as if I would have implemented this on my own after the corresponding feature request was posted...

I'll maybe give this another try, but likely not very soon, thanks again.

Reply:
Understandable and totally fine for me.
Will close this PR and create a feature request instead the next time.
Sorry if I wasted your time and nevertheless thanks for this nice tool and sharing it to the public.